integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java

```diff
@@ -375,12 +377,21 @@ public static boolean verifyRollingRestartOccurred(Map<String, OffsetDateTime> p
    */
   public static Callable<Boolean> domainStatusReasonMatches(oracle.weblogic.domain.Domain domain,
                                                             String statusReason) {
+    LoggingFacade logger = getLogger();
     return () -> {
       if (domain != null && domain.getStatus() != null && domain.getStatus().getReason() != null) {
+        logger.info("DEBUG: domain status reason: {0}", domain.getStatus().getReason());
```

Comment:
Remove DEBUG string since you are not using any debug flag.

Reply:
fixed